geomesa-dynamo/pom.xml

```diff
@@ -1,27 +1,40 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <parent>
-        <artifactId>geomesa</artifactId>
         <groupId>org.locationtech.geomesa</groupId>
+        <artifactId>geomesa</artifactId>
         <version>1.2.1-SNAPSHOT</version>
     </parent>
     <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>geomesa-cassandra</artifactId>
-    <name>GeoMesa Cassandra</name>
+    <artifactId>geomesa-dynamo</artifactId>
+    <name>GeoMesa Dynamo</name>
     <packaging>pom</packaging>
 
     <modules>
+        <module>geomesa-dynamo-core</module>
+        <module>geomesa-dynamodb-datastore</module>
         <module>geomesa-cassandra-datastore</module>
     </modules>
 
     <properties>
+        <aws-sdk.version>1.10.62</aws-sdk.version>
         <cassandra.version>3.0.0</cassandra.version>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     </properties>
 
     <dependencyManagement>
         <dependencies>
             <dependency>
+                <groupId>com.amazonaws</groupId>
+                <artifactId>aws-java-sdk-bom</artifactId>
+                <version>${aws-sdk.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
```

Comment:
this is the recommended way to do the imports for the aws java sdk (it is in Amazon's readme https://github.com/aws/aws-sdk-java/). the added benefit is that with this is I don't need to explicitly exclude all the other modules in the sdk as I would have to otherwise.